graylog2-web-interface/src/pages/LoginPage.jsx

```diff
@@ -92,6 +113,11 @@ const LoginPage = () => {
         <LoginPageStyles />
         {formatLastError()}
         {renderLoginForm()}
+        {hasCustomLogin && !isCloud && (
+          <StyledButton as="a" onClick={() => setUseFallback(!useFallback)}>
+            {`Login with ${useFallback ? loginComponent.type.replace(/^\w/, (c) => c.toUpperCase()) : 'default method'}`}
```

Comment:
A small thing with this is that it will show up while we log the user in using Okta. Right now I have no quick fix for it, so I'll create an issue for it.